src/style/sections/blog.css

```diff
@@ -0,0 +1,49 @@
+.blog {
+    margin-top: 104px;
+}
+
+/* features  */
+
+.features {
+    margin-bottom: 104px;
+}
+
+.features-frame {
+    display: flex;
+    gap: 24px;
+}
+
+.company:first-child {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+}
+
+.company:last-child {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+}
+
+.company-row {
+    display: flex;
+    gap: 24px;
+}
+
+/* ----------------------------------------------- */
+
+.gallery {
+    margin-top: 104px;
+}
+
+.gallery-photo {
```

Comment:
should combine to this block `.gallery-photo, .photo { display: flex;
    gap: 24px;
}`